api-specifications/admin-api/admin-api-2.2.0.yaml

```diff
@@ -1,6 +1,7 @@
 openapi: 3.0.1
 info:
   version: 2.2.0
+  title: "Ed-Fi Admin API"
```

Comment:
@stephenfuqua 
I remember you told me to remove the title, are you ok to include it again?

Reply:
Without the title swagger editor https://editor.swagger.io/ wouldn't load the metadata